docs/business-apps/power-automate/sharepoint-connector-actions-triggers.md

```diff
@@ -4,237 +4,205 @@ ms.date: 03/13/2020
 ---
 
 # Microsoft SharePoint Connector in Power Automate
-
 Microsoft SharePoint Connector in Power Automate supports the following flow triggers and actions.
```

Comment:
In Power Automate, Microsoft SharePoint Connector supports the following flow triggers and actions.